.github/workflows/pipeline-main.yml

```diff
@@ -12,6 +12,18 @@ jobs:
     uses: ./.github/workflows/pipeline-core.yml
   other:
     uses: ./.github/workflows/pipeline-other.yml
+  docs:
+    needs:
+      - clients
+      - core
+      - other
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: ./.github/ci/publish_docs.sh
+      env:
+        GITHUB_SHA: ${{GITHUB_SHA}}
```

Comment:
Is it not `${{ github.sha }}`?

Reply:
Switched it to be based off the release version instead.